schema/backwards_compatibility_test.go

```diff
@@ -91,7 +91,7 @@ func TestBackwardsCompatibilityImageIndex(t *testing.T) {
          "platform": {
             "architecture": "arm",
```

Comment:
I understand that "arm" is what gets expressed by GOARCH, which is why it is used here.  But would it not be better to disambiguate the property by calling it "arm32"?